device-types/QNAP/TS-432PXU.yaml

```diff
@@ -0,0 +1,38 @@
+---
+manufacturer: QNAP
+model: TS-432PXU
+slug: qnap-ts-432pxu
+part_number: TS-432PXU
+u_height: 1
+airflow: front-to-rear
+front_image: true
+rear_image: true
+is_full_depth: true
+subdevice_role: parent
+device-bays:
+  - name: Disk 1
+  - name: Disk 2
+  - name: Disk 3
+  - name: Disk 4
```

Comment:
Disks are not currently being modelled in this repository.

Reply:
Addressed in https://github.com/netbox-community/devicetype-library/pull/2521/commits/67cdbb458c786b68d02badc850ebf40b2c00a1be